pyccel/extensions/STC_Extensions/Set_extensions.h

```diff
@@ -0,0 +1,18 @@
+#ifndef SET_EXTENSIONS_H
+#define SET_EXTENSIONS_H
+
+
+#define _c_MEMB(name) c_JOIN(i_type, name)
+
+static inline i_key _c_MEMB(_pop)(i_type* self) {
+    _c_MEMB(_iter) itr = _c_MEMB(_begin)(self);
+    if (itr.ref)
+    {
+        i_key value = *(itr.ref);
+        _c_MEMB(_erase_at)(self, itr);
+        return value;
+    }
+    return *(itr.ref);
+}
+
```

Comment:
This is not an external extension. Please move it to `internals/` with the other methods that only exist inside Pyccel code

Reply:
Should i create a new folder named `internals`?

Reply:
Ah sorry the folder is called `stdlib`

Reply:
Fixed : cade789